shard-operator/src/main/resources/application.properties

```diff
@@ -31,4 +31,6 @@ event-bridge.webhook.account-id=${WEBHOOK_TECHNICAL_ACCOUNT_ID:402cf429-da04-493
 %openshift.quarkus.kubernetes.deployment-target=openshift
 %openshift.quarkus.container-image.build=true
 %openshift.event-bridge.k8s.platform=openshift
-event-bridge.shard-id=${EVENT_BRIDGE_SHARD_ID:150b23e9-0b34-45b7-a91a-5ec388d03a1d}
\ No newline at end of file
+event-bridge.shard-id=${EVENT_BRIDGE_SHARD_ID:150b23e9-0b34-45b7-a91a-5ec388d03a1d}
+
+quarkus.log.category."com.redhat.service.bridge.shard.operator".level=DEBUG
```

Comment:
log level should be set at runtime, otherwise you enable `debug` by default and this might not be what we want. If it is useful for you during the development, I suggest you to use `%dev.quarkus.log.category."com.redhat.service.bridge.shard.operator".level=DEBUG` instead

Reply:
Agreed.

Reply:
Done.